### DISCUSSION TITLE
Can't Change Price Check Back

### DISCUSSION THREAD

**Discussion by zChybo** [SnosMe/awakened-poe-trade#1070] [Q&A]
So I changed the price check hotkey just to mess around and play around with the features since i've  been using it for so long but never really got to know the program. Long story short I changed the hot key to price check to Ctrl + V I thought I remembered how I changed it but when I tried to set it back to CTRL + D  it wont let me, (You can see it on the top right)    
![image](https://github.com/SnosMe/awakened-poe-trade/assets/9813120/b9c184ca-1f9f-4147-b236-01b57690a316)

**Comment by SnosMe** [comment]
> it wont let me

Can you be more specific as to what exactly prevents you to change hotkey?
